plugins/single_plugins/place.cpp

```diff
@@ -0,0 +1,106 @@
+#include <view.hpp>
+#include <core.hpp>
+#include <workspace-manager.hpp>
+#include <signal-definitions.hpp>
+
+using namespace std;
+
+class wayfire_place_window : public wayfire_plugin_t
+{
+	signal_callback_t created_cb;
+	signal_callback_t workarea_changed_cb;
+	int cascade_x, cascade_y;
+
+	public:
+	void init(wayfire_config *config)
+	{
+		created_cb = [=] (signal_data *data)
+		{
+			auto view = get_signaled_view(data);
+
+			if (view->role != WF_VIEW_ROLE_TOPLEVEL ||
+				view->parent ||view->fullscreen ||
+				view->maximized)
+				return;
+
+			auto workarea = output->workspace->get_workarea();
+			auto section = config->get_section("place");
+			auto placement_mode = section->get_option("mode", "center");
```

Comment:
just a minor thing, you get get the section + option in `init()` , and save the placement_mode as `wf_option`, but it's fine now as well

Reply:
I don't know what you mean by this.

Reply:
Just that you can store the result of get_option() as a member variable, but this isn't a big problem